demosplan/DemosPlanCoreBundle/Logic/Procedure/ProcedureService.php

```diff
@@ -834,6 +836,8 @@ public function addProcedureEntity(array $data, string $currentUserId): Procedur
             $blueprintId = $data['copymaster'] ?? null;
             $blueprintId = $blueprintId instanceof Procedure ? $blueprintId->getId() : $blueprintId;
             $newProcedure = $this->setAuthorizedUsersToProcedure($newProcedure, $blueprintId, $currentUserId);
+            $newProcedure = $this->setPlanningOffices($newProcedure, $blueprintId, $currentUserId);
```

Comment:
this looks weird. Is it correct?

Reply:
I agree, it looks weird and maybe it can be solved differently. Let me explain the context:

a) When a procedure is created, by default the PlaningOffices and AuthorizedUsers are copied from the Blueprint. We cannot do that in this case, as the orga that is creating the procedure should be the only one that can access it. This pr sets the PlaningOffices as the current orga as long as the user has the permission feature_admin_new_procedure.
b) If the Support Role of step 1 removes the permission of the orga to creare Procedures, the Orga should still be able to keep editing the created Procedures (that is why it is important to do a) point.